src/connection-manager/update-amendments-from-json.ts

```diff
@@ -0,0 +1,63 @@
+import 'dotenv/config'
+import * as fs from 'fs'
+
+import { rippleTimeToUnixTime } from 'xrpl'
+
+import { query } from '../shared/database/utils'
+import { AmendmentEnabled } from '../shared/types'
+import logger from '../shared/utils/logger'
+
+const log = logger({ name: 'database-agreement' })
+const filePath = 'src/shared/data/amendments_enabled.json'
+
+interface AmendmentEnabledJson {
+  networks: string
+  amendments: Array<{
+    id: string
+    networks: string
+    ledger_index: number
+    tx_hash: string
+    date: number
```

Comment:
The date should be different on every network.  This data model makes it seem like the date is shared.